ingestclient/client.py

```diff
@@ -229,6 +229,15 @@ def main(configuration=None, parser_args=None):
 
     if args.cancel:
         # Trying to cancel
+        boss_backend_params = {"client": {
```

Comment:
You've hardcoded the backend properties.   We'll need to pull them from the json file passed in on the commandline.